SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -277,10 +288,9 @@ void USpatialNetDriver::OnAcceptingPlayersChanged(bool bAcceptingPlayers)
 			// Extract map name and options
 			FWorldContext* WorldContext = GEngine->GetWorldContextFromWorld(GetWorld());
 			check(WorldContext);
-			
-			FURL RedirectURL = FURL(&WorldContext->LastURL, *GlobalStateManager->DeploymentMapURL, (ETravelType)WorldContext->TravelType);
```

Comment:
The constructor explicitly parses the options of the second parameter `FURL(&WorldContext->LastURL, *GlobalStateManager->DeploymentMapURL, (ETravelType)WorldContext->TravelType);` in this case the `*GlobalStateManager->DeploymentMapURL`.
If you skip the constructor I believe the options will not be respected and thus you'll lose the options specified in the `DeploymentMapURL`

I think the problem is options that were used in the client travel being dropped when connecting to a server with a different map? 
If so we should update the `DeploymentMapURL` to have the correct options.

Reply:
Add a comment to indicate this constructor is to keep the URL options.